lib/Service/FileService.php

```diff
@@ -63,6 +64,7 @@ public function __construct(
         private readonly IGroupManager   		$groupManager,
         private readonly ISystemTagManager      $systemTagManager,
         private readonly ISystemTagObjectMapper $systemTagMapper,
+        private readonly ObjectEntityMapper     $objectEntityMapper,
```

Comment:
Why is this added here? I don't see any code being added in this service that uses this mapper